spec/identity_doc_auth/lexis_nexis/error_generator_spec.rb

```diff
@@ -0,0 +1,233 @@
+require 'spec_helper'
+
+RSpec.describe IdentityDocAuth::LexisNexis::ErrorGenerator do
+
+  let(:exception_notifier) { instance_double('Proc') }
+
+  let(:i18n) do
+    FakeI18n.new(
+      'doc_auth.errors.lexis_nexis.barcode_content_check',
+      'doc_auth.errors.lexis_nexis.barcode_read_check',
+      'doc_auth.errors.lexis_nexis.birth_date_checks',
+      'doc_auth.errors.lexis_nexis.control_number_check',
+      'doc_auth.errors.lexis_nexis.expiration_checks',
+      'doc_auth.errors.lexis_nexis.full_name_check',
+      'doc_auth.errors.lexis_nexis.general_error_no_liveness',
+      'doc_auth.errors.lexis_nexis.general_error_liveness',
+      'doc_auth.errors.lexis_nexis.id_not_verified',
+      'doc_auth.errors.lexis_nexis.multiple_back_id_failures',
+      'doc_auth.errors.lexis_nexis.ref_control_number_check',
+      'doc_auth.errors.lexis_nexis.selfie_failure',
+      )
+  end
+  let(:config) do
+    IdentityDocAuth::LexisNexis::Config.new(
+      i18n: i18n,
+      exception_notifier: exception_notifier,
+    )
+  end
+
+  def build_alerts(*failed)
+    {
+      passed: [],
+      failed: failed,
+    }
+  end
+
+  def build_error_info (doc_auth_result, alerts, pm_result)
```

Comment:
1. nit: can we remove the space between the name and the paren?

2. WDYT of using keyword arguments? The random `nil` at the end of the various callsites was confusing. Or maybe we just add a default value? `pm_result = nil` so that we can simplify the callsites

Reply:
1. No problem, missed that.
2. Honestly, I thought about doing that and then just forgot in the churn of moving from idp to gem.